client/src/components/POISelect.js

```diff
@@ -35,6 +35,22 @@ const POISelect = (props) => {
         {
         label: '7',
         value: '7'
+        },
```

Comment:
I think this is good for now! I just added all the test POIs from Craig if you wanted to try populating the POI dropdown with those from the test data but that could be something for a future task